ballista/rust/scheduler/src/api/handlers.rs

```diff
@@ -163,20 +163,41 @@ pub(crate) async fn get_job_summary<T: AsLogicalPlan, U: AsExecutionPlan>(
         })),
     }
 }
+
 /// Generate a dot graph for the specified job id and return as plain text
 pub(crate) async fn get_job_dot_graph<T: AsLogicalPlan, U: AsExecutionPlan>(
     data_server: SchedulerServer<T, U>,
     job_id: String,
 ) -> Result<String, Rejection> {
-    let graph = data_server
+    let maybe_graph = data_server
```

Comment:
I liked just "graph" :cry:

Reply:
I refactored this to avoid the need for this intermediate variable